src/containers/SelectionBar.jsx

```diff
@@ -24,15 +24,15 @@ const SelectionBar = ({ t, selected, selectedCount, album, onHide, onAddToAlbum,
     <span className={styles['coz-selectionbar-separator']} />
     <button
       disabled={selectedCount === 0}
-      className={styles['pho-action-album-add']}
+      className={styles['coz-action-album-add']}
```

Comment:
true. but it was already in Cozy-UI, so I made the app use it instead of rewriting the style. :/